microsoft-edge/webview2/index.md

```diff
@@ -3,106 +3,74 @@ description: Host web content in your Win32 app with the Microsoft Edge WebView
-
-## Debugging WebView2
-
-### DevTools
-
-You can use [Microsoft Edge (Chromium) Developer Tools](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium) to debug web content displayed in WebView, just as you would in the browser. While having focus on the WebView window, press `F12`, or press `Ctrl` + `Shift` + `I`, or Right Click + choose `Inspect` to open Developer Tools.
-
-:::image type="complex" source="./media/f12.png" alt-text="F12":::
-   F12
-:::image-end:::  
-
-<!--![F12](./media/f12.png)  -->  
-
-**Note when debugging application in Visual Studio with the native debugger attached, `F12` may trigger the native debugger instead of Developer Tools. Use `Ctrl` + `Shift` + `I`, or Right Click + `Inspect` to avoid potential hotkey conflict.**
-
-### Visual Studio
-
-You can use the script debugger in Visual Studio 2019 (minimum version 16.4 Preview 2) to debug your script within WebView2 right from the IDE. Make sure the **JavaScript diagnostics** component in **Desktop development with C++** workload is installed.
-
-:::image type="complex" source="./media/vs-js-diagnostics.jpg" alt-text="Visual Studio JavaScript diagnostics":::
-   Visual Studio JavaScript diagnostics
-:::image-end:::  
-
-<!--![vs-js-diagnostics](./media/vs-js-diagnostics.jpg)  -->  
-
-Right click on your project and select **Properties**. Under **Configuration Properties** > **Debugging** > **Debugger Type**,  choose the **JavaScript (WebView2)** option to enable WebView2 script debugging. More details to follow soon.
+## Getting Started
+To build and test your application using the WebView2 control, you need to have both [Microsoft Edge (Chromium)](https://www.microsoftedgeinsider.com/download/) and the [WebView2 SDK](https://aka.ms/webviewnuget)installed. Select one of the options below to get started! 
+- [Getting Started with Win32 C/C++]()
+- [Getting Started with WPF]()
+- [Getting Started with WinForms]()
 
-:::image type="complex" source="./media/vs-script-debugger.jpg" alt-text="Visual Studio script debugger":::
-   Visual Studio script debugger
-:::image-end:::  
+The [WebView2 Samples](https://github.com/MicrosoftEdge/WebView2Samples) repository contains samples that demonstrate all of the WebView2 SDK's features and their API use patterns. As we add more features to the WebView2 SDK, we will regularly update our sample applications.
+- [Win32 C/C++ Sample]()
+- [WPF Sample]()
+- [WinForms Sample]()
 
-<!--![vs-script-debugger](./media/vs-script-debugger.jpg)  -->  
+## Next Steps
+For more thorough information on how to build and deploy WebView2 application checkout our conceptual documentation and how-to guides.
 
-### Visual Studio Code
+### Concepts
+1. [WebView2 SDK and Microsoft Edge Versioning]()
+2. [Distributing WebView2 Applications]()
+3. [Utilizing and Disabling Microsoft Edge Browser Features in WebView2]()
```

Comment:
TODO: add links to published docs when they go live.

Reply:
Updated.